frontend/src/components/NavigationBar.tsx

```diff
@@ -0,0 +1,58 @@
+import { Link, useLocation } from "react-router-dom";
+import {
+  AccountCircleOutlined,
+  LaptopOutlined,
+  WebOutlined,
+} from "@mui/icons-material";
+import { Container, Divider, List, ListItem } from "@mui/material";
+
+import LogoImage from "../assets/images/Logo/Logo.png";
+
+import { DrawerButton } from "./DrawerButton";
+
+export const NavigationBar = () => {
+  const location = useLocation();
+  const pages = ["Account", "Dashboard", "PeerPrep"];
+  return (
+    <Container
+      disableGutters
+      sx={{ backgroundColor: "primary.light", height: "100vh" }}
+    >
+      <Container component="img" src={LogoImage} sx={{ my: "10%" }} />
+      <Divider />
+      <List>
+        {pages.map((page) => (
+          <ListItem
```

Comment:
We should wrap this in a `NavLink` and have it handle the selectedness of a button.